lib/galaxy_test/selenium/test_tool_form.py

```diff
@@ -89,7 +89,6 @@ def _table_to_key_value_elements(self, table_selector):
 
     @selenium_test
     def test_rerun(self):
-        self.use_legacy_history()
         self._run_environment_test_tool()
         self.history_panel_wait_for_hid_ok(1)
         self.hda_click_primary_action_button(1, "rerun")
```

Comment:
The reason why this test fails, is that the `rerun` button is expected in the legacy context, not because the rerun does not work. In my opinion it would be better to redirect these tests to use the new history.

Reply:
then you want to fix that, this is my point

Reply:
@jmchilton thx, yea that would work. So my plan was to create individual PRs for all the tests now using `use_legacy_history` and convert them individually but I think there are different ideas here.

Reply:
This is fine, but the beta tests should be in failing status until you've done so. And I don't see that issue being tracked either.

Reply:
Rerun test fixed in: https://github.com/galaxyproject/galaxy/pull/13721/commits/6f94353d18b42b9916bd9aa349c0b0f5cf11b6b5